apps/aecore/test/aecore_suite_utils.erl

```diff
@@ -257,6 +264,10 @@ mine_blocks_until_txs_on_chain_loop(_Node, _TxHashes, 0, _Acc) ->
 mine_blocks_until_txs_on_chain_loop(Node, TxHashes, Max, Acc) ->
     case mine_blocks_loop(1, key) of
         {ok, [Block]} -> %% We are only observing key blocks
+          % set a properly huge key block mining interval
+          ok = rpc:call(
+              Node, application, set_env, [aecore, expected_mine_rate,
```

Comment:
Is this mine_blocks_until_txs_on_chain_loop changing expected_mine_rate during same test? Would this potentially lead to inconsistent DB?